lib/StyleDictionary.js

```diff
@@ -380,17 +390,51 @@ export default class StyleDictionary extends Register {
 
   /**
    * @param {string} platform
-   * @returns {Promise<TransformedTokens>}
+   * @param {{ cache?: boolean }} [opts]
    */
-  async exportPlatform(platform) {
-    await this.hasInitialized;
-
+  getPlatformConfig(platform, opts) {
     if (!platform || !this.platforms?.[platform]) {
-      throw new Error('Please supply a valid platform');
+      throw new Error(`Please supply a valid platform, "${platform}" does not exist`);
```

Comment:
See earlier comment in test file about combining two messages doing the same thing into one